Code/GraphMol/MolDraw2D/DrawMol.cpp

```diff
@@ -141,9 +142,13 @@ void DrawMol::finishCreateDrawObjects() {
 // ****************************************************************************
 void DrawMol::initDrawMolecule(const ROMol &mol) {
   drawMol_.reset(new RWMol(mol));
-  if (!isReactionMol_ &&
-      (drawOptions_.prepareMolsBeforeDrawing || !mol.getNumConformers())) {
-    MolDraw2DUtils::prepareMolForDrawing(*drawMol_);
+  if (!isReactionMol_) {
+    if (drawOptions_.prepareMolsBeforeDrawing) {
+      MolDraw2DUtils::prepareMolForDrawing(*drawMol_);
+    } else if (!mol.getNumConformers()) {
+      const bool canonOrient = true;
```

Comment:
I'm a little bit unclear of the difference here, but if there are no conformers, wouldn't you still want to call prepareMolForDrawing?  I'm not seeing the use case for just making coords without calling prepare.

Reply:
We don't want to call `prepareMolForDrawing()` because it does a bunch of stuff other than just adding coordinates: kekulizing, adding chiral Hs, wedging bonds, etc. We shouldn't ignore the caller's request to skip that stuff.